Mlem/App/Views/Root/Tabs/Settings/InteractionBarEditor/InteractionBarWidgetPickerView.swift

```diff
@@ -32,20 +33,23 @@ struct InteractionBarWidgetPickerView<Configuration: InteractionBarConfiguration
                 }
             }
         }
+        .toolbar {
+            ToolbarItem(placement: .topBarTrailing) {
+                CloseButtonView {
+                    dismiss()
+                }
+            }
+        }
```

Comment:
Could do with less padding at the top imo
```suggestion
        }
        .contentMargins(.top, 0)
```